src/openforms/conf/base.py

```diff
@@ -629,10 +629,16 @@
     "LOG_OUTGOING_REQUESTS_MAX_AGE", default=7
 )  # number of days
 
+# TODO: convert to feature flags so that newly deployed instances get the new behaviour
+# while staying backwards compatible for existing instances
 USE_LEGACY_DIGID_EH_OIDC_ENDPOINTS = config(
     "USE_LEGACY_DIGID_EH_OIDC_ENDPOINTS",
     default=True,
 )
+USE_LEGACY_ORG_OIDC_ENDPOINTS = config(
```

Comment:
Is it intentional that these feature flags being `True` only adds `DeprecationWarning`s? Can't we also exclude the actual urlpatterns if they are `False`?

Reply:
Yeah it's intentional - three reasons:

* in Django, you want to defer settings module evaluation, so accessing settings at the module-level is a no-no (mozilla-django-oidc's `urls.py` setup is terrible for that), because...
* using `@override_settings(...)` in tests only patches the settings, but it doesn't reload the URL conf, unless you specify a different `ROOT_URLS`, and at  that point we're no longer testing our actual URL conf but some hypothetical thing
* having both the new and legacy endpoints makes it easier to transition to the new situation - modifying the Redirect URIs int he OIDC Provider then doesn't require you to re-deploy the application with the updated envvars at the same time

Reply:
FYI - they also return different URL names to reverse, it's not just a deprecation warning